DataSynapseApi/ArmoniK.DevelopmentKit.GridServer.Client/Properties.cs

```diff
@@ -194,6 +210,8 @@ public string ConnectionString
       {
         try
         {
+          if (string.IsNullOrEmpty(value)) return;
```

Comment:
If the value of the connection string is empty and you are going back in the code without it, there are no problem?